.github/workflows/main.yml

```diff
@@ -112,6 +112,13 @@ jobs:
         env:
           LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
 
+      # Run Wafer Lighthouse audit (from any push)
+      - name: Wafermap Performance Audit with Lighthouse CI
+        if: env.HAS_LHCI_GITHUB_APP_TOKEN == 'true' && github.event_name == 'push'
+        run: npm run wafer-performance
+        env:
+         LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
```

Comment:
I'll defer to @rajsite on whether or not this is appropriate.

Reply:
I deleted this part, now the wafer will be tested once with the other tests using npm run performance